pkg/kubectl/describe.go

```diff
@@ -1195,7 +1195,9 @@ func (d *ReplicaSetDescriber) Describe(namespace, name string, describerSettings
 
 	running, waiting, succeeded, failed, err := getPodStatusForController(pc, selector)
 	if err != nil {
-		return "", err
+		if !errors.IsNotFound(err) {
```

Comment:
The normal pattern is to output this as part of the result, not to print warnings.